src/thinblock.cpp

```diff
@@ -1101,6 +1101,36 @@ void SendXThinBlock(CBlock &block, CNode *pfrom, const CInv &inv)
     pfrom->blocksSent += 1;
 }
 
+bool IsThinBlockValid(const CNode* pfrom, const std::vector<CTransaction>& vMissingTx, const CBlockHeader& header)
+{
+    // Check that that there is at least one txn in the xthin and that the first txn is the coinbase
+    if(vMissingTx.size() <= 0)
+    {
+        LogPrintf("No Transactions found in thinblock or xthinblock %s from peer %s (id=%d).\n",
+            header.GetHash().ToString(), pfrom->addrName.c_str(), pfrom->id);
+        return false;
+    }
+    if(!vMissingTx[0].IsCoinBase())
```

Comment:
A perfect 50% of the ifs have a space before the brace, the others don't... Consistency is useful to have :)